migrations/versions/0316_uploads_for_all.py

```diff
@@ -0,0 +1,41 @@
+"""
+
+Revision ID: 0316_uploads_for_all
+Revises: 0315_document_download_count
+Create Date: 2019-05-13 10:44:51.867661
+
+"""
+from alembic import op
+from app.models import UPLOAD_LETTERS
+
+
+revision = '0316_uploads_for_all'
+down_revision = '0315_document_download_count'
+
+
+def upgrade():
+    op.execute("""
+        INSERT INTO
+            service_permissions (service_id, permission, created_at)
+        SELECT
+            id, '{permission}', now()
+        FROM
+            services
+        WHERE
+            NOT EXISTS (
+                SELECT
+                FROM
+                    service_permissions
+                WHERE
+                    service_id = services.id and
+                    permission = '{permission}'
+           )
+    """.format(
+        permission=UPLOAD_LETTERS
+    ))
+
+
+def downgrade():
+    op.execute("DELETE from service_permissions where permission = '{}'".format(
+        UPLOAD_LETTERS
+    )
```

Comment:
this will delete everything, including stuff previously added manually. i think i'd rather no possible downgrade for this migration i think?

Reply:
Yeah I was in 2 minds about what the right way to go there was.

Updated to `pass` in 6677e32